webhooks/assign-class-label/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: assign-class-label-webhook
+  labels:
+    app: assign-class-label
```

Comment:
If you remove your labels in `metadata.labels` and `spec.template.metadata.labels`, and you remove `spec.selector`, and set them in `kustomization.yaml` instead (...which you are already doing), kustomize will take care of setting everything correctly.